microcosm_pubsub/result.py

```diff
@@ -146,7 +146,7 @@ def log(self, logger, opaque):
             self.result.level,
             entry,
             exc_info=self.exc_info,
-            extra=dict(**self.extra, **opaque.as_dict()),
+            extra={**opaque.as_dict(), **self.extra},
```

Comment:
I'm switching the order so that we end up with the values that were explicitly set (from `extra`) rather than the ones implicitly defined (from `opaque`).